.github/workflows/ci.yml

```
@@ -0,0 +1,185 @@
+          - platform: linux/amd64
+            tag-suffix: amd64
+          - platform: linux/arm64
+            tag-suffix: arm64
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set deployment environment
+        id: set-env
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:19:38: Double quote to prevent globbing and word splitting

(shellcheck)